### PR TITLE
Konvert EnumConstantElement

### PR DESCRIPTION
**Pull request by jrodbx**
depends on https://github.com/square/wire/pull/872

### CODE REVIEW

File: wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/EnumConstantElement.kt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  val tag: Int = 0,
+  val documentation: String = "",
+  val options: List<OptionElement> = emptyList()
+) {
+
+  fun toSchema() = buildString {
+    appendDocumentation(this, documentation)
+    append("$name = $tag")
+
+    if (!options.isEmpty()) {
```
Comment (JakeWharton) [inline wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/EnumConstantElement.kt]:
```suggestion
    if (options.isNotEmpty()) {
```

File: wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/EnumConstantElement.kt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ */
+package com.squareup.wire.schema.internal.parser
+
+import com.squareup.wire.schema.Location
+import com.squareup.wire.schema.internal.Util
+import com.squareup.wire.schema.internal.Util.appendDocumentation
+
+data class EnumConstantElement(
+  val location: Location,
+  val name: String? = null,
```
Comment (JakeWharton) [inline wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/EnumConstantElement.kt]: Hmm another null name. From auto convert or actually needed?

File: wire-schema/src/test/java/com/squareup/wire/schema/internal/parser/EnumElementTest.kt [hunk 3]
```diff
@@ -142,40 +131,37 @@ class EnumElementTest {
...
-        .build()
+    val value = EnumConstantElement(
+        location = location,
+        name = "NAME",
+        tag = 1,
+        documentation = "Hello"
+    )
     val expected = """
-        |// Hello
+      |// Hello
```
Comment (JakeWharton) [inline wire-schema/src/test/java/com/squareup/wire/schema/internal/parser/EnumElementTest.kt]: 👀

File: wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/EnumConstantElement.kt (hunk 2 above)
Comment (jrodbx) [reply wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/EnumConstantElement.kt]: Auto convert.  Reverted.

**Pull request merged**